app/tests/system/setup/setup.js

```diff
@@ -0,0 +1,142 @@
+/*
+ * Loop through the custom contexts that exist when the app starts
+ * and grab their names.
+ *
+ * To change the environment we're testing against, specify it in site.json.
+ * This should align with what the app is built against, which is declared in
+ * app/environment.js hostUrl
+ */
+
+var Site = require('../site.json');
+
+var Setup = {};
+
+var selectors = {
+    previewButton: '#authorize',
+    registeredAuctionList: '.js-auction-lists'
+};
+
+var expected = {
+    //EXAMPLE Webviews constructed for project.
+    'noInternet': /no-internet\.html/i,
+    'couldNotLoad': /could-not-load\.html/i
+};
+
+Setup.appPreview = function(browser) {
+    browser.contexts(function(result) {
+        browser
+            .session(function(type) {
+                browser
+                    .log('Running on platform - ' + type.value.platformName);
+                if (type.value.platformName === 'Android') {
```

Comment:
Ok. What if we still created a method and passed `platform` or `platformName` into that method. Then in the method you compare it by forcing the incoming value to all lowercase:
```javascript
var isAndroid = function(platform) {
    platform = platform || 'unknown';
    return platform.toLowerCase() === 'android';
}
```
As we hit other inconsistencies we can capture those in the method. That gives us a consistent method we always use to check if we're on Android or iOS (two methods).